Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -83,19 +84,17 @@ const Contest = () => {
 
     const renderTimeRemaining = useCallback(
         () => {
-            const { endTime } = contest || {};
-
-            if (!endTime) {
+            if (!remainingTimeInMilliseconds) {
                 return null;
             }
-
-            const duration = convertToSecondsRemaining(new Date(endTime));
-
+            
+            const currentSeconds=remainingTimeInMilliseconds/1000;
```

Comment:
Spaces

Reply:
There is a ESLint modification for spaces: "ESLint: Expected blank line before this statement.(padding-line-between-statements)"

Reply:
There is a ESLint modification for spaces: ESLint: Expected blank line before this statement.(padding-line-between-statements)